network_wrangler/ProjectCard.py

```diff
@@ -7,7 +7,7 @@
 from jsonschema import validate
 from jsonschema.exceptions import ValidationError
 from jsonschema.exceptions import SchemaError
-from .Logger import WranglerLogger
+from Logger import WranglerLogger
```

Comment:
This doesn't work for me in the docker dev environment. Does it work for you in the dev environment? 

What I get:

```bash
docker build wrangler-dev
docker run wrangler-dev /bin/bash
docker-compose run wrangler-dev python -c "import network_wrangler"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/jovyan/work/network_wrangler/__init__.py", line 7, in <module>
    from .ProjectCard import ProjectCard
  File "/home/jovyan/work/network_wrangler/ProjectCard.py", line 10, in <module>
    from Logger import WranglerLogger
ModuleNotFoundError: No module named 'Logger'
```

Changing this lint to `from .Logger` fixes it.